### DISCUSSION TITLE
Github readme-stats takes too long to update

### DISCUSSION THREAD

**Discussion by Soumilgit** [anuraghazra/github-readme-stats#3759] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

In my profile,the no. of issues and contributions doesn't update soon,tried using  https://github-readme-stats-ruby-one.vercel.app/  instead of  https://github-readme-stats.vercel.app/ but then it gives error in readme display.Please resolve.

### Describe the solution you'd like

The stats should get updated within atleast 1-2 hours,will like if someone helps with removal of caching.

**Comment by qwerty541** [comment, marked as answer]
Hey @Soumilgit.

The default cache time on our public deployment is 6 hours. We use caching to decrease consumption of GitHub API tokens points in order to avoid public deployment downtimes. If you want your card to be updated more often you can [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#on-vercel) using your own GitHub API token and set there any cache time using `CACHE_SECONDS` environment variable.

https://github-readme-stats-ruby-one.vercel.app/ - Is not the official deployment, it is own deployment of some user. Any member of our team do not have an access to it and cannot do anything.
